weave/andrew_init/global_state.py

```diff
@@ -0,0 +1,27 @@
+# forget contexts, let's just use globals for now and then revisit
```

Comment:
Not sure if we are still using contextvars.  Either way, we should make sure all shared state is in one place that's easy to see...